include/quic/stream.hpp

```diff
@@ -119,7 +119,7 @@ namespace oxen::quic
         bool ready{false};
         int64_t _stream_id;
 
-        Connection& get_conn();
+        const Connection& get_conn() const { return conn; }
```

Comment:
We should probably have *both* of these (so that you get a const/non-const reference according to whether you have a const/non-const Stream on which you're calling it).